packages/atclient/src/atclient.c

```diff
@@ -239,3 +245,144 @@ void atclient_free(atclient *ctx) {
   atclient_connection_free(&(ctx->root_connection));
   atclient_connection_free(&(ctx->secondary_connection));
 }
+
+int atclient_get_encryption_key_shared_by_me(atclient *ctx, atclient_atsign *recipient, char *enc_key_shared_by_me)
```

Comment:
I can't tell whats an input parameter and whats an output parameter, make sure to prefix input only pointers with `const <type>* <name>`